spectrocloud/schemas/backup_policy.go

```diff
@@ -0,0 +1,71 @@
+package schemas
+
+import "github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+func BackupPolicySchema() *schema.Schema {
+	return &schema.Schema{
+		Type:     schema.TypeList,
+		Optional: true,
+		MaxItems: 1,
+		Elem: &schema.Resource{
+			Schema: map[string]*schema.Schema{
+				"prefix": {
+					Type:        schema.TypeString,
+					Required:    true,
+					Description: "Prefix for the backup name. The backup name will be of the format <prefix>-<cluster-name>-<timestamp>.",
+				},
+				"backup_location_id": {
+					Type:        schema.TypeString,
+					Required:    true,
+					Description: "The ID of the backup location to use for the backup.",
+				},
+				"schedule": {
+					Type:        schema.TypeString,
+					Required:    true,
+					Description: "The schedule for the backup. The schedule is specified in cron format. For example, to run the backup every day at 1:00 AM, the schedule should be set to 0 1 * * *.",
+				},
+				"expiry_in_hour": {
+					Type:        schema.TypeInt,
+					Required:    true,
+					Description: "The number of hours after which the backup will be deleted. For example, if the expiry is set to 24, the backup will be deleted after 24 hours.",
```

Comment:
@nikchern is there a max and min value for this field?

Reply:
Not documented so it should be defined by the data type.